src/screens/Notification.js

```diff
@@ -133,11 +133,20 @@ return (
         <Property name="toast">
           <Description>
             A boolean flag that indicates whether or not the Notification is a
-            Toast Notification.
+            Toast Notification. When using Toast Notification a position is
+            avalible to pass in for where the toast will show.
```

Comment:
I would put the info about position in the description under the PropertyValue object section instead of the general description for toast since it only applies when passing an object to the toast prop